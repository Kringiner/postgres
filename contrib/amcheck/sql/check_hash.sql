
SELECT setseed(1);

-- Test that index check works
CREATE TABLE hash_test AS SELECT (random()* 10000 + 1) num from generate_series(1,10000);
CREATE INDEX hash_idx ON hash_test USING hash(num);
SELECT hash_index_check('hash_idx');