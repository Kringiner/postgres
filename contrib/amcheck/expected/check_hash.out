SELECT setseed(1);
 setseed 
---------
 
(1 row)

-- Test that index check works
CREATE TABLE hash_test AS SELECT (random()* 10000 + 1) num from generate_series(1,10000);
CREATE INDEX hash_idx ON hash_test USING hash(num);
SELECT hash_index_check('hash_idx');
 hash_index_check 
------------------
 
(1 row)

